Resources/Public/JavaScript/pxa_dealers.js

```diff
@@ -123,6 +129,32 @@
                         convertJq.hide(overlay[0]);
                     }
 
+                    /**
+                     * Display only items that has dealers
+                     * @private
+                     */
+                    function _showVisibleItemsRadioBox() {
```

Comment:
Not nice solution to copy/paste a code, needs some refactoring along with _showVisibleItemsCheckBox.